src/Model/State.re

```diff
@@ -9,21 +9,26 @@ open Oni_Input;
 open Oni_Syntax;
 
 module Ext = Oni_Extensions;
-module ContextMenu = Oni_Components.ContextMenu;
 module KeyDisplayer = Oni_Components.KeyDisplayer;
 module Completions = Feature_LanguageSupport.Completions;
 module Diagnostics = Feature_LanguageSupport.Diagnostics;
 module Definition = Feature_LanguageSupport.Definition;
 module LanguageFeatures = Feature_LanguageSupport.LanguageFeatures;
 module BufferSyntaxHighlights = Feature_Editor.BufferSyntaxHighlights;
 
+module ContextMenu = {
+  type t =
+    | NotificationStatusBarItem
+    | Nothing;
+};
+
```

Comment:
I think we might want to have a `Feature_ContextMenu` project for the application-specific model and store, but for now I just plopped the model in here. If you agree this is the right approach I can include that in this PR as well though.

Reply:
That makes sense to me to have a `Feature_ContextMenu`. I'm not sure if it is pressing right now since the surface area is small.

I'll leave it up to you if you want to include it in this PR, or in a separate one down the road 👍

Reply:
We'll do it later then.